### PR TITLE
[python] Add an option to Analysis to fix parameters to a set of values

### PR DESCRIPTION
**Pull request by mreboud**

### CODE REVIEW

**Comment by mreboud** [comment]
Ah sorry, I realize that the description is not updated

File: python/eos/analysis.py [hunk 1]
```diff
@@ -81,11 +83,18 @@ def __init__(self, priors, likelihood, global_options={}, manual_constraints={})
         eos.debug('manual_constraints:')
         for cn, ce in manual_constraints.items():
             eos.debug(' - {name}'.format(name=cn))
+        eos.debug('fixed_parameters:')
+        for cn, ce in fixed_parameters.items():
```
Comment (dvandyk) [inline python/eos/analysis.py:87]: Should that not me ``pn, pv`` as in parameter name, parameter value?

**Review by dvandyk — APPROVED** [review]
Mostly OK, wouldn't mind merging it. However, see one comment on weird naming of iterator elements.

@mreboud: merge if you can't be bothered :smile:

File: python/eos/analysis.py (hunk 1 above)
Comment (mreboud) [reply python/eos/analysis.py:87]: It definitely should :)

**Pull request merged**